cmd/gaia/app/benchmarks/txsize_test.go

```diff
@@ -25,11 +26,14 @@ func ExampleTxSendSize() {
 		Inputs:  []bank.Input{bank.NewInput(addr1, coins)},
 		Outputs: []bank.Output{bank.NewOutput(addr2, coins)},
 	}
-	sig, _ := priv1.Sign(msg1.GetSignBytes())
-	sigs := []auth.StdSignature{{nil, sig, 0, 0}}
-	tx := auth.NewStdTx([]sdk.Msg{msg1}, auth.NewStdFee(0, coins...), sigs, "")
+	fee := auth.NewStdFee(gas, coins...)
+	signBytes := auth.StdSignBytes("example-chain-ID",
+		1, 1, fee, []sdk.Msg{msg1}, "")
+	sig, _ := priv1.Sign(signBytes)
+	sigs := []auth.StdSignature{{nil, sig}}
+	tx := auth.NewStdTx([]sdk.Msg{msg1}, fee, sigs, "")
 	fmt.Println(len(cdc.MustMarshalBinaryBare([]sdk.Msg{msg1})))
 	fmt.Println(len(cdc.MustMarshalBinaryBare(tx)))
 	// output: 80
-	// 167
```

Comment:
Previously the benchmark masked the true tx size, since it omitted account number, sequence number, and gas (as they were the default value). This increase is due to gas, but removal of accnum and sequence number will increase space savings.